content/300-accelerate/600-faq.mdx

```diff
@@ -8,6 +8,18 @@ toc: true
 
 <TopBlock>Below are frequently asked questions about Accelerate.</TopBlock>
 
+## When do I have to enable static IP for Prisma Accelerate?
+
+A static IP address is an IPv4 or an IPv6 address that doesn’t change. Unlike dynamic IP addresses, which can change unpredictably, traffic from static IP addresses are predictable. The predictability of traffic originating from static IP addresses allows you to precisely identify traffic sources. IP addresses need to be predictable when implementing security measures such as IP allowlisting or firewall configurations to restrict database access to trusted sources only.
+
+![What is a static IP](./images/static-ip.png)
+
+If your database is using IP allowlisting, enable static IP within your Accelerate configuration in your project environment and then add the IP address you obtained in your database firewall's IP allowlist, ensuring secure access for Accelerate while blocking unauthorized requests to your database.
```

Comment:
```suggestion
If your database is using IP allowlisting, enable static IP within your Accelerate configuration in your project environment and then add the IP address you obtained in your database firewall's IP allowlist, ensuring secure access from Accelerate while blocking unauthorized requests to your database.
```

Reply:
Can we simplify this section with;

For databases using IP allowlisting, enable static IP for Accelerate in your specified project environment, and add the obtained IP address to your database's allowlist to ensure authorized access from Accelerate

Reply:
I like the simplified version a lot 🙌 .

Reply:
```suggestion
For databases using IP allowlisting, enable static IP for Accelerate in your specified project environment, and add the obtained IP address to your database's allowlist to ensure authorized access from Accelerate.
```